PhysicsTools/PatUtils/python/tools/runMETCorrectionsAndUncertainties.py

```diff
@@ -499,12 +499,14 @@ def produceMET(self, process,  metType, metModuleSequence, postfix):
             configtools.cloneProcessingSnippet(process, getattr(process,"producePatPFMETCorrections"), postfix, noClones = noClonesTmp, addToTask = True)
             addToProcessAndTask(_myPatMet,  getattr(process,'patPFMet').clone(), process, task)
             getattr(process, _myPatMet).metSource = cms.InputTag("pfMet"+postfix)
-            getattr(process, _myPatMet).srcPFCands = self.getvalue("pfCandCollection")
+            getattr(process, _myPatMet).srcPFCands = copy.copy(self.getvalue("pfCandCollection"))
```

Comment:
I thought that these cases do not require a copy. (this is a dict .value)
Am I wrong?

Reply:
I guess I'm wrong
https://github.com/cms-sw/cmssw/blob/4546ca54b1ef88845cbc585619642f4b9e16ed83/PhysicsTools/PatAlgos/python/tools/ConfigToolBase.py#L90-L93
IMHO, this method would be the right place to return a copy